src/widgets/bottomModals/BottomCurrencyEditor.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable import/prefer-default-export */
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { TextInput, StyleSheet } from 'react-native';
+
+import { BottomModalContainer } from './BottomModalContainer';
+import { OnePressButton } from '../OnePressButton';
+import globalStyles from '../../globalStyles';
+
+const {
+  modalTextInput,
+  modalText,
+  modalButtonText,
+  modalOrangeButton,
+  button,
+  buttonText: buttonTextStyle,
+} = globalStyles;
+
+export const BottomCurrencyEditor = React.memo(
```

Comment:
I think this would be better overall as `CurrencyEditor` - just like `TextEditor` where you can wrap it in either the `BottomModalContainer` or the `ModalContainer`. All it would take would be to remove the `BottomModalContainer` and wrap this component in it. Better to do that even if it doesn't really work out of the box imo